src/components/api-reference/schemas/measurements.js

```diff
@@ -54,7 +54,18 @@ const STRATA_FIELDS = [
   {name: 'eligiblePopulation', value: 'integer', description: 'The total number of eligible patients as described by the measure. Must be greater than or equal to zero. In the measures specifications, this field is also referred to as "Eligible Population Numerator/Denominator".', notes: 'writable'},
   {name: 'stratum', value: 'string', description: 'The strata associated with the performance rate measurement.', notes: 'writable'},
   {name: 'reportingRate', value: 'float', description: 'The reporting rate, ranging from zero to one-hundred and representing a percentage, is equal to ((performanceMet + eligiblePopulationExclusion + eligiblePopulationException + performanceNotMet) / eligiblePopulation) * 100.'},
-  {name: 'performanceRate', value: 'float', description: 'The performance rate for a multiple performance rate measurement is calculated based on the overallAlgorithm of the corresponding measure. Currently, a measure\'s overallAlgorithm can be a <b>simpleAverage</b>: ((sum of performanceMet from all of the non-overall strata) / (sum of performanceMet and performanceNotMet from all of the non-overall strata)) / (number of non-overall strata), <b>weightedAverage</b>: ((sum of performanceMet from all of the non-overall strata) / (sum of performanceMet and performanceNotMet from all of the non-overall strata)) * 100, or <b>sumNumerators</b>: the sum of the performanceMet property from each non-overall stratum.'}
+  {
+    name: 'performanceRate',
+    value: 'float',
+    description: 'Performance rates are stored and represented as percentages with values from 0.00 to 100.00. ' +
+      'The performance rate for a multiple performance rate measurement is calculated using the overallAlgorithm of the corresponding measure. ' +
+      'Note, performance rate calculations exclude any "overall" strata. ' +
+      'Also note, for the <b>simpleAverage</b> algorithm, each stratum\'s performance rate is calculated as <samp>performanceMet / (performanceMet + performanceNotMet - exclusions)</samp>. ' +
+      'Currently, a measure\'s overallAlgorithm may be one of the following:<ul>' +
+      '<li><b>simpleAverage:</b> <samp>(sum strata\'s performance rates) / (number of strata) * 100</samp>,</li>' +
+      '<li><b>weightedAverage:</b> <samp>((sum strata\'s performanceMet) / (sum strata\'s performanceMet and performanceNotMet)) * 100</samp>, or,</li>' +
+      '<li><b>sumNumerators:</b> <samp>sum strata\'s performanceMet</samp>.</li></ul>'
```

Comment:
There is also the overallStratumOnly algorithm which calculates only based on the "overall" stratum.

Reply:
Good call, added in https://github.com/CMSgov/qpp-submissions-docs/pull/83/commits/eacc7376cdbf6a6c03164795c0d81765501a7384